tests/test_catalog.py

```diff
@@ -1141,6 +1141,7 @@ def test_validate_all(self, cat: Catalog) -> None:
             cat.normalize_hrefs("/tmp")
         cat.validate_all()
 
+    def test_validate_all_invalid(self) -> None:
```

Comment:
drive by, but this was part of a parameterized test but this section didn't depend at all on the parameterized value